README.md

```diff
@@ -67,8 +67,11 @@ The following projects are official OpenJS Foundation projects. If you are inter
 | | Project        | Charter           | Contributing       | Code of Conduct       |
 |-|----------------|:-----------------:|:------------------:|:---------------------:|
 | <img alt="logo of project ajv" src="https://github.com/openjs-foundation/artwork/raw/HEAD/projects/ajv/ajv-logo-color.svg" width="30" />  | [Ajv](https://ajv.js.org/) |   | [Contributing Guide](https://ajv.js.org/contributing) | [Code of Conduct](https://github.com/ajv-validator/ajv/blob/HEAD/CODE_OF_CONDUCT.md) |
+| <img alt="logo of project Jest" src="https://raw.githubusercontent.com/facebook/jest/main/website/static/img/jest.svg" width="30" />  | [Jest](https://jestjs.io/) |   | [Contributing Guide](https://github.com/facebook/jest#contributing) | [Code of Conduct](https://github.com/facebook/jest#code-of-conduct) |
 | <img alt="logo of project JSON Schema" src="https://github.com/openjs-foundation/artwork/raw/HEAD/projects/jsonschema/jsonschema-logo-color.svg" width="30" />  | [JSON Schema](https://json-schema.org/) |   | [Contributing Guide](https://github.com/json-schema-org/json-schema-spec/blob/draft-next/CONTRIBUTING.md) | [Code of Conduct](https://github.com/json-schema-org/.github/blob/HEAD/CODE_OF_CONDUCT.md) |
```

Comment:
```suggestion
| <img alt="logo of project JSON Schema" src="https://github.com/openjs-foundation/artwork/raw/HEAD/projects/jsonschema/jsonschema-logo-color.svg" width="30" />  | [JSON Schema](https://json-schema.org/) |   | [Contributing Guide](https://github.com/json-schema-org/json-schema-spec/blob/HEAD/CONTRIBUTING.md) | [Code of Conduct](https://github.com/json-schema-org/.github/blob/HEAD/CODE_OF_CONDUCT.md) |
```

This link is broken because of this. https://github.com/json-schema-org/json-schema-spec/pull/1251